cmd/subnetcmd/deploy.go

```diff
@@ -119,14 +119,31 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 	case models.Local:
 		app.Log.Debug("Deploy local")
 		deployer := subnet.NewLocalSubnetDeployer(app)
-		if err := deployer.DeployToLocalNetwork(chain, chainGenesis); err != nil {
+		subnetID, blockchainID, err := deployer.DeployToLocalNetwork(chain, chainGenesis)
+		if err != nil {
 			if deployer.BackendStartedHere() {
 				if innerErr := binutils.KillgRPCServerProcess(app); innerErr != nil {
 					app.Log.Warn("tried to kill the gRPC server process but it failed: %w", innerErr)
 				}
 			}
+			return err
 		}
-		return err
+		sc, err := app.LoadSidecar(chain)
+		if err != nil {
+			return fmt.Errorf("creation of chains and subnet was successful, but failed to load sidecar for update: %w", err)
+		}
+		if sc.Networks == nil {
+			sc.Networks = make(map[string]models.NetworkData)
+		}
+		sc.Networks[models.Local.String()] = models.NetworkData{
+			SubnetID:     subnetID,
+			BlockchainID: blockchainID,
+		}
+		if err := app.UpdateSidecar(&sc); err != nil {
```

Comment:
is this update of sidecar needed for local network?

Reply:
It is not _needed_, but I thought it's awkward that we write this info when deploying to fuji, but not when local. So I concluded it should be consistent and therefore we should write it.